types/pouchdb-adapter-cordova-sqlite/index.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for pouchdb-adapter-websql 6.1
+// Project: https://pouchdb.com/, https://github.com/pouchdb/pouchdb
```

Comment:
Please link to [the specific package](https://github.com/pouchdb-community/pouchdb-adapter-cordova-sqlite) on this line.

Reply:
Sure, have checked in changes just now